dls-controls/malcolmjs
# Added sequence diagrams for some of the most common operations

## Description

Added sequence diagrams for common operations:
- Connecting to the websocket
- Getting block details
- Loading a layout
- Running a method

## Testing instructions

Add a set up instructions describing how the reviewer should test the code
- Review code
- Check Travis build
- Review changes to test coverage
- Build docs and view the sequence diagram page

## Agile board tracking

connect to #195 
closes #195
